enclave-manager/web/src/components/enclaves/configuration/EnclaveConfigurationForm.tsx

```diff
@@ -98,7 +98,7 @@ export const EnclaveConfigurationForm = forwardRef<
         {},
       );
 
-    onSubmit({ ...data, args: newArgs });
+    onSubmit({enclaveName: "", restartServices: false, ...data, args: newArgs });
```

Comment:
I am not sure this is needed - these fields should already be supplied by `ConfigureEnclaveModal` (ie they should never be `null` or undefined in the supplied `data` object)

Reply:
interesting, no I don't get this same issue when I run yarn run build.

In the interest of expediency, I'm not particularly interested in working out what the difference is atm.